### DISCUSSION TITLE
Topologies/Domains Multiple api Manager with Same groupId

### DISCUSSION THREAD

**Discussion by Rawane** [Axway-API-Management-Plus/apigateway-openlogging-elk#219] [Q&A]
![POC-LOG](https://user-images.githubusercontent.com/7751218/176406840-29bfa4e9-20b5-481d-a52d-dc59df31c060.png)
Hi,
We have several domains as shown in the diagram, my question how to configure the apm builder with the same groupId(group-1).
Another question How to set username/password for each APM.
Sample
secrets: 
    apimgrUsername: "apiadmin1"
    apimgrPassword: "changeme1"
secrets: 
    apimgrUsername: "apiadmin2"
    apimgrPassword: "changeme2"

**Comment by cwiechmann** [comment]
You need to use the region feature, as part of the configuration. For instance:
`group-1|FR-C1|https://(API-Manager of FR-C1 Group1):8075`

I believe you need a default API-Manager configured as well. This is used to lookup users, when they are using the API-Gateway Traffic-Monitor.

To your second question: You can configured only one API-Manager user, which is used for all configured API-Managers.
https://github.com/Axway-API-Management-Plus/apigateway-openlogging-elk/blob/b734b7ddce613ba98dc7304dbc3e287f5f644e9c/env-sample#L87

**Comment by Rawane** [comment]
The APM in C1 and APM in C2 are the same group-id (group-1)
**group-1**|FR-C1|https://(API-Manager of FR-C1 Group1):8075,**group-1**|FR-C2|https://(API-Manager of FR-C1 Group1):8075

**Reply by cwiechmann** [reply]
It looks almost correct, but I think there is a typo:
**group-1|FR-C1**|https://(API-Manager of **FR-C1 Group1**):8075,**group-1|FR-C2**|https://(API-Manager of **_FR-C2_ Group1**):8075
